packages/manager/modules/pci/src/projects/project/data-processing/onboarding/onboarding.constants.js

```diff
@@ -0,0 +1,16 @@
+export const GUIDES = [
+  {
+    id: 'overview',
+    link: 'https://docs.ovh.com/gb/en/data-processing/overview/',
```

Comment:
Only one link, no links depending on the subsidiary ?

Reply:
We are only in EU during the beta phase with documentation in EU_GB only